docs.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
```

Comment:
Given that this generates docs, I'd rename it go `generate-docs.sh`

Reply:
I think this should probably be a bazel rule in some way?